config/secrets.js

```diff
@@ -59,3 +57,17 @@ module.exports.updateFetching = function(flag, cb) {
     return cb(err);
   });
 };
+
+// update settings
+module.exports.update = function(type, settings, cb) {
+  cb = cb || function() {};
+
+  for (var key in settings) {
+    var item = type + ':' + key;
+    nconf.set(item, settings[key]);
+  }
+
+  nconf.save(function(err) {
+    return cb(err);
+  });
+};
```

Comment:
Yes, it is temporary.

Toggling fetching also depends on this. We may want to move that setting to the secrets.json and eventually to the database as well.